docs/README.md

```diff
@@ -34,13 +38,15 @@ By hovering on the thumbnail of an Item, you can either click on it to enter the
 If you want to create annotations for am Item, click on it to access its page.
 
 ## Annotating an Item
```

Comment:
Since we have a same-level header called "Image Comparison", maybe we could rename this header into something similar, like "Basic Campaign - Annotating an Item"?